### ISSUE TITLE
Check whitespace between words/phrases

### ISSUE THREAD

**Issue by Fleker** [adeshpande3/Facebook-Messenger-Bot#20]
Hey I started training my bot last night. Looking at some of the items in the wordlist and in the training printouts, I noticed that some of the items appear to be two words concatenated together, without spaces. I wonder if there's a bug in the Facebook parser that is not adding spaces everywhere necessary.

It seems to happen when running `python createDataset.py` and getting the `conversationData.txt`.

**Comment by Fleker** [comment]
I'm looking in my conversation data again and noticing this happening in several places. When one person's message ends, it doesn't seem like there is proper spacing for the response.

* "pityyou"
* "snowi"
* "soulmake"

**Comment by anne1994** [comment]
@Fleker were you able to resolve the issue?

**Comment by Fleker** [comment]
No. I think I need to fix an issue in that Python script, but I haven't gotten around to it.
